projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -90,11 +90,19 @@ public function enqueue_admin_scripts() {
 				'textdomain' => 'jetpack-social',
 			)
 		);
+
+		global $publicize;
+		wp_localize_script(
```

Comment:
Turns out I gave you outdated advice here (sorry). See next comment on what we should do instead.